protected.go

```diff
@@ -54,28 +69,71 @@ func New(protect Protect, dnsServerIP string) *Protector {
 // Resolve resolves the given address using a DNS lookup on a UDP socket
 // protected by the given Protect function.
 func (p *Protector) Resolve(network string, addr string) (*net.TCPAddr, error) {
-	op := ops.Begin("protected-resolve").Set("addr", addr)
-	defer op.End()
-	conn, err := p.resolve(op, network, addr)
-	return conn, op.FailIf(err)
+	switch network {
+	case "tcp", "tcp4", "tcp6":
+	case "udp", "udp4", "udp6":
```

Comment:
Why are we allowing udp here?

Reply:
We shouldn't -- I think that was a leftover from when I was debugging. Just removed that line.

Reply:
Oh, got it, could be either